From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Andy Wingo <wingo@pobox.com>
Date: Fri, 13 Mar 2020 17:28:43 +0100
Subject: [PATCH] Revert "Fix build on platforms where the stack grows
 upwards."

This patch was only meant to apply to the 3.0 branch.
---
 libguile/continuations.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libguile/continuations.c b/libguile/continuations.c
index c02598b..80914bc 100644
--- a/libguile/continuations.c
+++ b/libguile/continuations.c
@@ -302,7 +302,7 @@ scm_dynthrow (SCM cont)
 
 #if SCM_STACK_GROWS_UP
   if (dst + continuation->num_stack_items >= &stack_top_element)
-    grow_stack (cont, mra);
+    grow_stack (cont);
 #else
   dst -= continuation->num_stack_items;
   if (dst <= &stack_top_element)
